bzkanban.js

```diff
@@ -1062,32 +1079,48 @@ function dropCard(ev) {
     }
 }
 
-function dropBacklog(ev) {
-    if (ev.target.classList.contains("drop-target")) {
-        ev.target.classList.remove("drop-target-hover");
-    }
-    ev.preventDefault();
+function showBacklog() {
+    var button = document.getElementById("btnShowBacklog");
+    var backlogCol = document.querySelector("#BACKLOG.board-column");
+
+    if (backlogCol.style.display === "none" || null) {
+        // Load backlog on first access.
+        var backlog = backlogCol.querySelector(".cards");
+        if (backlog.children.length == 0) {
+            loadBacklogCards();
```

Comment:
In 868dd250d489f5f4b5cea943e18a7c494ae91948, this has been bypassed. Was going to remove unless you want to refactor.